boards/DCM/Src/Io/Io_LSM6DS33.c

```diff
@@ -0,0 +1,58 @@
+#include <App_SharedExitCode.h>
+#include "Io_LSM6DS33.h"
+#include "App_Imu.h"
+
+// The I2c slave device addresses to use when reading/writing to/from the IMU
+// See table 11 in section 6.1.1 of the LSM6DS33 data sheet for details
+// Register Address Constants
+#define FUNC_CFG_ACCESS 0x01
+#define ORIENT_CFG_G 0x0B
+#define INT1_CTRL 0x0D
+#define INT2_CTRL 0x0E
+#define CTRL1_XL 0x10
+#define CTRL2_G 0x11
+#define CTRL3_C 0x12
+#define CTRL4_C 0x13
+#define CTRL6_C 0x15
+#define CTRL7_C 0x16
+#define CTRL8_XL 0x17
+#define CTRL9_XL 0x18
+#define CTRL10_C 0x19
+#define STATUS_REG 0x1E
+#define DATA_OUT_STRT 0x22
+#define IMU_READ_ADDR 0xd7
+#define IMU_WRITE_ADDR 0xd6
+
+struct LSM6DS33
+{
+    I2C_HandleTypeDef *imu_i2c_handle;
+    // Acceleration in m/s^2
+    float accel_x;
+    float accel_y;
+    float accel_z;
+};
+
+static struct LSM6DS33 lsm_6ds33;
+
+ExitCode Io_LSM6DS33_ExtiCallback()
+{
+        lsm_6ds33.accel_x = 0;
+        lsm_6ds33.accel_y = 0;
+        lsm_6ds33.accel_z = 0;
+    return EXIT_CODE_OK;
```

Comment:
Also the exit code returned here isn't being used inside the callback function. I think you just make this a void func for now

Reply:
It's used in App_SetPeriodicCanSignals_Imu as a check before sending CAN signals

Reply:
Actually I think I'll just make the ExitCode into a static variable in the Io struct

Reply:
The idea is that we aren't responsible for calling the function at all so it shouldn't be called in App_SetPeriodicSignals_Imu. It should only be called in the ISR (Cube already configured this for u) when the interrupt is triggered.

Reply:
I have some ideas for what we can do but for this PR lets keep it simple for IO stuff

Reply:
Yeah I realized after I was calling it twice when there was no need to. It is gone now